### ISSUE TITLE
Set up CI

### ISSUE THREAD

**Issue by gchers** [AdaLogics/paper-analyser#9]
Tasks:
- verify that every PR adheres to coding standards (e.g., use `black`).
- run tests to verify that every PR doesn't break stuff.

**Comment by DavidKorczynski** [comment]
I like this but I would delay this until we have removed all the existing code and only have alexandria code. Are you okay with this? If so, we can assign a milestone here.

**Comment by gchers** [comment]
Yes, absolutely. No point in making it before having the pipeline ready-ish.
